controller/cmd/destination/main.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func main() {
-	addr := flag.String("addr", "127.0.0.1:8089", "address to serve on")
+	addr := flag.String("addr", ":8086", "address to serve on")
```

Comment:
Is there a reason we are no longer using `127.0.0.1`? I am sure it's not a big deal but just curious

Reply:
Yeah, I didn't make this change at first and it took me hours to figure out what was wrong.

This causes the destination server to listen on all interfaces instead of just localhost.  If it only listens on localhost then the proxy can't connect to it.

Reply:
Ah, I get it. Thanks!